### ISSUE TITLE
using the junit formatter doesnt work

### ISSUE THREAD

**Issue by wickett** [gauntlt/gauntlt#72]
using --format junit doesnt work because underneath gauntlt, cucumber wants a --dir passed to it.

**Comment by karebear90** [comment]
any work arounds for this?

**Comment by Gaurang033** [comment]
I am also facing the same issue, I need to generate the JUnit xml, however if I run the command without --out option it gives following error. 

You *must* specify --out DIR for the junit formatter

If i provide --out parameter, then I get following error. 
Error: unknown argument '--out'.

would someone please let me know how to generate the JUnit report

**Comment by mjgorman** [comment]
+1 Have the same issue.

**Comment by frameloss** [comment]
I realize this doesn't really contribute to the fix of this issue, but for those held-up by this issue there is an easy workaround. In your working directory create the file './config/cucumber.config' and add the --out setting there ... for example:

```
default: --out /var/tmp/gauntlt/junit
```

I found that I had to use a fully-qualified path, but that is an easy problem to work around in your CI jobs. Hope that helps.
